### DISCUSSION TITLE
Tổ chức cấu trúc file phía backend

### DISCUSSION THREAD

**Discussion by khoavukhoa3004** [khoavukhoa3004/DADN_TTNT#4] [Q&A]
Chúng ta nên tổ chức folder dạng: controllers/, models/, routes/ hay kiểu với mỗi tính năng, vd như: Home sẽ có folder Home/, trong đó chứa cả 3 file controller.js, models.js, routes.js ??

**Comment by nguyencharlie** [comment]
Ý kiến cá nhân của tui thì vẫn nên để thành 3 thư mục là controllers/, models/, routes/, vì hiện tại thì tui cũng đang theo cấu trúc đó :v
Thực ra thì để kiểu 3 file controller.js, models.js, routes.js cũng được.
Nói chung theo tui nghĩ là mình nên tạo một poll để bình chọn cho dễ á :v
